.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: node_js
 node_js:
+  - 8
   - 5
```

Comment:
while we are at it, lets remove 5 and add 6 and 9. we should have all LTS versions and the latest stable node version